### PR TITLE
Remove unneeded Redundancies

### PR DESCRIPTION
**Pull request by PartTimeJS**
Removes redundant getUser and getGuilds calls to speed up login by a second or two.

### CODE REVIEW

File: src/routes/discord.js [hunk 1]
```diff
@@ -38,14 +38,14 @@ router.get('/callback', catchAsyncErrors(async (req, res) => {
         //const client = DiscordClient.instance;
```
Comment (clburlison) [inline src/routes/discord.js:38]:
```suggestion
```

File: src/routes/discord.js [hunk 2]
```diff
@@ -38,14 +38,14 @@ router.get('/callback', catchAsyncErrors(async (req, res) => {
...
         DiscordClient.setAccessToken(response.data.access_token);
         const user = await DiscordClient.getUser();
-        const guilds = await DiscordClient.getGuilds();
+        //const guilds = await DiscordClient.getGuilds();
```
Comment (clburlison) [inline src/routes/discord.js:41]:
```suggestion
```

File: src/routes/discord.js [hunk 3]
```diff
@@ -38,14 +38,14 @@ router.get('/callback', catchAsyncErrors(async (req, res) => {
...
 
         req.session.logged_in = true;
         req.session.user_id = user.id;
         req.session.username = `${user.username}#${user.discriminator}`;
-        const perms = await DiscordClient.getPerms();
+        const perms = await DiscordClient.getPerms(user);
         req.session.perms = perms;
-        req.session.guilds = guilds;
+        //req.session.guilds = guilds;
```
Comment (clburlison) [inline src/routes/discord.js:48]:
```suggestion
```

File: src/services/discord.js [hunk 4]
```diff
@@ -76,7 +76,7 @@ class DiscordClient {
             weather: false,
             devices: false
         };
-        const user = await this.getUser();
+        //const user = await this.getUser();
```
Comment (clburlison) [inline src/services/discord.js:79]:
```suggestion
```

**Review by clburlison — CHANGES_REQUESTED** [review]
Can we just delete these lines if they aren't being used? Can always add them back in later if we need to process the guilds in an additional way.

File: src/routes/discord.js (hunk 1 above)
Comment (PartTimeJS) [reply src/routes/discord.js:38]: I deleted them in my fork, but I figured I would let Versx make that call or keep them there for future use.

**Comment by sketchysocks** [comment]
Works great! Much faster logins <3

**Pull request merged**